calico-cloud/image-assurance/scanners/registry-scanner.mdx

```diff
@@ -53,7 +53,7 @@ To deploy the registry scanner as a pod in Kubernetes cluster, we recommend that
 
 ### Download the registry scanner
 
-The registry scanner comes in a Docker image. To get the image, run this command: `docker pull quay.io/tigera/image-assurance-registry-scanner:{{imageassuranceversion}}`.
+The registry scanner comes in a Docker image. To get the image, run this command: `docker pull quay.io/tigera/image-assurance-registry-scanner:v1.15.3`.
```

Comment:
I think it's worth adding a ToDo comment just to mark down that we're explicitly coming back here to switch it to the `imageassuranceversion` parameter on the next release.

Reply:
@WilliamTigera Best to revert changes to this doc. That way, this version mismatch will stay in production only for CC 19.2. We'll be cutting the new version for the next release, 19.3, which incorporates CE 3.19ep2.

Reply:
Reverted.